src/app/content/components/Page.spec.tsx

```diff
@@ -34,6 +34,7 @@ import PageNotFound from './Page/PageNotFound';
 import allImagesLoaded from './utils/allImagesLoaded';
 
 jest.mock('./utils/allImagesLoaded', () => jest.fn());
+jest.mock('./utils/attachHighlight', () => jest.fn());
```

Comment:
Is this still needed? I thought I removed that part that caused it to be called

Reply:
idk? I just merged master in and didn't change this file in my last commit.

Reply:
yea but this was added before when we were messing around karina, it might need to be removed now